MAUI/ProgressBar/getting-started.md

```diff
@@ -0,0 +1,249 @@
+
+### Register the handler
+
+Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
+
+{% highlight c# tabtitle="~/MauiProgram.cs" hl_lines="17" %}
+
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace MyProject
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+
+## Adding namespace
+
+Add the following namespace.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar"
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.ProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Initializing progress bar
+
+Create an instance for the progress bar control, and add it as content. The progress bar control has two variants: SfLinearProgressBar and SfCircularProgressBar. Each renders the progress in its own shape, such as rectangle and circle, respectively. Initialize both the progress bars with a progress value using the Progress property as demonstrated in the following code sample.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<!--Using linear progress bar-->
+<progressBar:SfLinearProgressBar Progress="75"/>
+
+<!--Using circular progress bar-->
+<progressBar:SfCircularProgressBar Progress="75"/>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+// Using linear progress bar. 
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar { Progress = 75 };
+this.Content = linearProgressBar;
+
+// Using circular progress bar.
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar { Progress = 75 };
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> By default, the value of progress should be specified between 0 and 100. To specify progress value between 0 and 1, specify the Minimum property to 0 and Maximum property to 1.
+
+Run the project, and check if you get following output to make sure that the project has been configured properly to add the progress bar.
+
+![.NET MAUI linear progress bar and circular progress bar](images/getting-started/progress-bar.png)
+
+## Enabling indeterminate state
+
+When the progress of a task cannot be shown determinately, you can enable the indeterminate state using the [IsIndeterminate]() property to know that any progress is happening in the background.
```

Comment:
When the progress of a task cannot be shown determinately, enable the indeterminate state using the [IsIndeterminate]() property to know if any progress is happening in the background.

Reply:
done